src/main/java/com/vention/stockmarket/exceptions/CustomFeignClientException.java

```diff
@@ -0,0 +1,16 @@
+package com.vention.stockmarket.exceptions;
+
+import lombok.Getter;
+
+@Getter
```

Comment:
if you are using Lombok you can @AllArgsConstructor instead of a manual constructor